src/collar/oc_relay.lsl

```diff
@@ -299,8 +299,15 @@ string HandleCommand(string sIdent, key kID, string sCom, integer iAuthed) {
         } else if ((lSubArgs!=[])==2) {
             string sBehav=llGetSubString(llList2String(lSubArgs,0),1,-1);
             list lTemp=llParseString2List(sBehav,[":"],[]);
-            if (g_iSmartStrip && llList2String(lTemp,0) == "remoutfit" && sVal == "force")
-                sBehav = "detachallthis:" + llList2String(lTemp,1);
+            if (g_iSmartStrip && llList2String(lTemp,0) == "remoutfit" && sVal == "force") {
+                if (llList2String(lTemp,1) != "") sBehav = "detachallthis:" + llList2String(lTemp,1);
+                else {  // only if  'remoutfit=force'
+                    list parts=["jacket","pants","shirt","shoes","skirt","socks","underpants","undershirt"]; //"gloves","alpha","tattoo","physics"];
+                    integer n;
+                    for (n=0; n<llGetListLength(parts); ++n)
+                        llMessageLinked(LINK_RLV,RLV_CMD,"detachallthis:"+llList2String(parts,n)+"=force",kID);

```

Comment:
See RLV API but this actually does support not specifying the part label
`@detachallthis[:<attachpt> or <clothing_layer>]=force (*)`
And based on this modified code, we're looping through all clothing layers, when we could just specify: `@detachallthis=force`

Reply:
@detachallthis=force  don't work!
RLV v3.2.1 / RLVa v2.2.0.58052RLV  don't execute  @detachallthis=force

Reply:
That would be a RLV bug if it does not work.  Since RLV specifications say it should.